resources/js/admin/users/components/SecurityLogsModal.vue

```diff
@@ -223,8 +229,8 @@ export default {
       return auxArraySorted;
     },
     /**
-     * Verify if value is a string o null
```

Comment:
don't remove the space, the code style for comments in JS, the * are aligned vertically
```
/**
 * Verify if value is a string o null
 */
isString(value) {
```